### PR TITLE
small addition

### PR DESCRIPTION
**Pull request by CerenB**
added one more function into `README` 
change the `scanner` to `mri` in a demo script
added some explanation related to abort into `getResponse.m` 

Question:  I am displaying the trigger count down in the monitor to the participant (for fun). See below. Can I add this snipped into my lib folder of CPP_PTB? Or is there another neat way to do such minor taste additions?         
```
%display the trigger count
displayInstr(['Experiment starting in ',num2str(4-triggerCounter),'...'],cfg);
```

### CODE REVIEW

**Comment by CerenB** [comment]
oh forgot to add/check with miss_hit.
Would it be nice to add that info (how to run in terminal, e.g. `mh_style functionxx.m`) into README file? Or it would be too confusing? ( I simply forgot the function name and had to go through the history in my terminal 👯 )

**Comment by Remi-Gau** [comment]
> Question: I am displaying the trigger count down in the monitor to the participant (for fun). See below. Can I add this snipped into my lib folder of CPP_PTB? Or is there another neat way to do such minor taste additions?

YES !! Actually maybe you can update the talkToMe subfunction of `waitForTrigger`or is that what you were suggesting?

**Comment by Remi-Gau** [comment]
> oh forgot to add/check with miss_hit.
> Would it be nice to add that info (how to run in terminal, e.g. `mh_style functionxx.m`) into README file? Or it would be too confusing? ( I simply forgot the function name and had to go through the history in my terminal dancers )

Opened a new issue for it. Github allows to easily reference a post and create an issue from it. :heart_eyes:

File: demos/CPP_waitForTriggerDemo.m [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cd ..;
 
-cfg.device = 'Scanner';
+cfg.device = 'mri';
```
Comment (Remi-Gau) [inline demos/CPP_waitForTriggerDemo.m:3]:
```suggestion
cfg.testingDevice = 'mri';
```

Comment (CerenB) [reply demos/CPP_waitForTriggerDemo.m:3]: should be!  
but the demo was crashing for another reason, cfg.MRI is not a defined field. And yet, it was called with isempty(cfg.MRI.xxx)

File: README.md [hunk 2]
```diff
@@ -173,11 +173,18 @@ This will handle the Eye Tracker (EyeLink set up) and can be called to initializ
...
+Counts a certain number of triggers coming from the mri/scanner before returning.
+Requires number of triggers to wait for.
+
+
 ## Annexes
 
 ### Experiment template [ WIP ]
 
-### `devSandobox.m` stand-alone
+### `devSandbox.m` stand-alone
```
Comment (marcobarilari) [inline README.md:187]:
```suggestion
### devSandbox.m stand-alone
```

I was waiting to change this in a repo like this, thanks :)

Comment (Remi-Gau) [reply README.md:187]: hahaha.

leaving typos around voluntarily to see if we spot them, are you? :stuck_out_tongue:

**Comment by Remi-Gau** [comment]
OK will merge this for now and fix the linter in our next PR

File: README.md (hunk 2 above)
Comment (marcobarilari) [reply README.md]: 🤪

**Pull request merged**